README.md

```diff
@@ -30,6 +30,39 @@ JCR hopper scripts comprise the following:
 - A list of hop configs that are run against a given node. Hops can also have their own descendant pipelines of hop configs.
 - A list of parameters that the script supports. Parameters always have a default value, thus passing arguments to the script runner is always optional. Each parameter also has a name, a script builder input type hint and an evaluation type that determines how the default values／arguments are to be interpreted.
 
+#### Creating a sample script
+
+JCR hopper sample scripts comprise the following:
+
+- A `label` that is displayed in the dropdown for Scripts.
+- A `log level` that determines which log messages are sent to the run handler.
+- A `list of hop configs` that are run against a given node. Hops can also have their own descendant pipelines of hop configs.
+- A `list of parameters` that the script supports. Parameters always have a default value, thus passing arguments to the script runner is always optional. Each parameter also has a name, a script builder input type hint and an evaluation type that determines how the default values／arguments are to be interpreted.
+
+##### Inside JCR Hopper repo
+
+Common reusable scripts could be added to JCR Hopper repo directly.
+
+- Create a new json file with `log level` , `list of hop configs` & `list of parameters` under `/apps/jcr-hopper/script-builder/scripts/samples/` folder.
+- Add an tag Entry in `/apps/jcr-hopper/script-builder/scripts/.content.xml` file for your new json and by defining user-friendly label for Sample script as `jcr:title`. Check this file on how it is added for previous sample script jsons.
```

Comment:
This is a specialized form of (“Inside application project repo”). I would prefer we document this only generically to avoid repetitions (we could add a section to `CONTRIBUTING.md` that details the specifics of adding a sample into this repo).

Also, I would describe the required state rather than the steps required to achieve them. We should assume the reader is competent and fluent in JCR. This means telling the user they should put the script as JSON to this path in JCR and tell them that if they want a custom title, they should add the `mix:title` mixin and set a `jcr:title`. If they don’t know how to do that with `.content.xml` files, they can do it in CRXde and export a package or they can look up one of the online resources that tells them how to do that.